packages/graphql/src/schema/mutation.ts

```diff
@@ -687,7 +679,7 @@ export const Mutation = builder.mutationType({
         },
       }),
 
-      createSession: asUser.field({
+      createSession: t.withAuth(asUserSessionExec).field({
```

Comment:
Do we want to allow users with session execution privileges to create and edit sessions? This might go beyond the scope of "execution"?